external/fv3fit/setup.py

```diff
@@ -41,7 +48,11 @@
     ],
     description="FV3Fit is used to train machine learning models.",
     install_requires=requirements,
-    dependency_links=["../loaders/", "../vcm/"],
+    dependency_links=[
+        "../loaders/",
+        "../vcm/",
+        "https://download.pytorch.org/whl/cu113",
```

Comment:
Is this something we should include as a hard dependency of fv3fit, or should we just modify our environment setup so we install this wheel on systems where we want to train on GPUs?

Reply:
I agree. We should probably just install basic torch and make sure its a version compatible with what we'd be installing with cuda on the docker image.